buffer/pom.xml

```diff
@@ -24,6 +24,7 @@
   </parent>
 
   <artifactId>netty-buffer</artifactId>
+  <version>siren-4.1.68-1-SNAPSHOT</version>
```

Comment:
Is the `siren-4.1.68-1` release made once the netty/arrow upgrade works in federate ?